src/quantuminspire/projectq/backend_qx.py

```diff
@@ -346,6 +346,39 @@ def _physical_to_simulated(self, physical_qubit_id: int) -> int:
 
         return physical_qubit_id
 
+    def _simulated_to_logical(self, simulated_qubit_id: int) -> int:
+        """
+        Return the logical qubit id given the allocated location on the simulated backend.
+
+        :param simulated_qubit_id: the allocated location on the simulated backend.
+
+        :return:
+            Logical bit position of simulated qubit.
+        """
+        # First step: simulated to physical
```

Comment:
optional: splitting this method into two named methods means we don't need to use comments

Reply:
I have split it